charts/argo-cd/templates/argocd-notifications/servicemonitor.yaml

```diff
@@ -2,12 +2,12 @@
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: {{ template "argo-cd.notifications.fullname" . }}-metrics
+  name: {{ template "argo-cd.notifications.fullname" . }}
   {{- if .Values.notifications.metrics.serviceMonitor.namespace }}
   namespace: {{ .Values.notifications.metrics.serviceMonitor.namespace }}
   {{- end }}
   labels:
-    {{- include "argo-cd.labels" (dict "context" . "component" .Values.notifications.name "name" .Values.notifications.name) | nindent 4 }}
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.notifications.name "name" "metrics") | nindent 4 }}
```

Comment:
This line was fine when I look to all other 4 service monitors in the chart.

Reply:
indeed it was, i reverted the change in c030308b9f6de7474a8e9280180c6b00b5d80fea